config/types.go

```diff
@@ -105,6 +105,8 @@ type CloudConfig struct {
 	Rancher           RancherConfig         `yaml:"rancher,omitempty"`
 	Runcmd            []yaml.StringandSlice `yaml:"runcmd,omitempty"`
 	Bootcmd           []yaml.StringandSlice `yaml:"bootcmd,omitempty"`
+	CloudInitcmd      []yaml.StringandSlice `yaml:"cloud_init_cmd,omitempty"`
```

Comment:
To be as compatible and harmonious with Bootcmd, Runcmd as possible, perhaps the yaml should read `cloud_initcmd` instead of `cloud_init_cmd`?

(This requires a change to `config/schema.go` in this PR, as well.)